src/main/java/hudson/plugins/promoted_builds/PromotionProcess.java

```diff
@@ -372,11 +379,21 @@ public Status isMet(AbstractBuild<?,?> build) {
         List<PromotionBadge> badges = new ArrayList<PromotionBadge>();
         for (PromotionCondition cond : conditions) {
             PromotionBadge b = cond.isMet(this, build);
-            if(b==null)
-                return null;
-            badges.add(b);
+            LOGGER.log(Level.INFO, "PromotionProcess.isMet(): any_met_condition={0}", this.any_met_condition);
```

Comment:
This logging does not help - PromotionProcess is not references in the entry

Reply:
i'll remove this line